scripts/docker/hub/Dockerfile

```diff
@@ -1,27 +1,40 @@
 FROM ubuntu:xenial
-LABEL MAINTAINER="Parity Technologies <devops-team@parity.io>"
 
-# install tools and dependencies
-RUN apt update && apt install -y --no-install-recommends openssl libudev-dev file curl jq
+# metadata
+ARG VCS_REF
+ARG BUILD_DATE
+
+LABEL io.parity.image.authors="devops-team@parity.io" \
+	io.parity.image.vendor="Parity Technologies" \
+	io.parity.image.title="parity/parity" \
+	io.parity.image.description="Parity Ethereum. The Fastest and most Advanced Ethereum Client." \
+	io.parity.image.source="https://github.com/paritytech/parity-ethereum/blob/master/scripts/docker/hub/Dockerfile" \
+	io.parity.image.revision="${VCS_REF}" \
+	io.parity.image.created="${BUILD_DATE}" \
+	io.parity.image.documentation="https://wiki.parity.io/Parity-Ethereum"
 
 # show backtraces
 ENV RUST_BACKTRACE 1
 
-# cleanup Docker image
-RUN apt autoremove -y \
-  && apt clean -y \
-  && rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/*
-
-RUN groupadd -g 1000 parity \
-  && useradd -m -u 1000 -g parity -s /bin/sh parity
+# install tools and dependencies
+RUN set -eux; \
+	apt-get update; \
+	apt-get install -y --no-install-recommends \
+	libudev-dev file curl jq; \
+# removed openssl
+# apt cleanup
+	apt-get autoremove -y; \
+	apt-get clean; \
+	rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/*; \
+# add user
+	groupadd -g 1000 parity; \
+	useradd -m -u 1000 -g parity -s /bin/sh parity
 
 WORKDIR /home/parity
 
-# add parity-ethereum to docker image
+# add parity-ethereum binary to docker image
 COPY artifacts/x86_64-unknown-linux-gnu/parity /bin/parity
 
-COPY scripts/docker/hub/check_sync.sh /check_sync.sh
```

Comment:
I don't know who put it there, but is it not needed anymore?

Reply:
returned, it was used somewhere, thanks.